src/events/DistanceEvents.ts

```diff
@@ -1,10 +1,17 @@
-export interface IPosition {
+/**
+ * The `IGeolocationPosition` interface describes a geolocation position.
```

Comment:
"Used by the {@link DistanceResourceComponent}" ?

I like pointing which interface is useful for which component.

cc: @fbeaudoincoveo

Reply:
Yes, I agree